esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -279,7 +275,49 @@ void loop()
         int len = SERIAL.read(buf, sizeof(buf));
         client.write(buf, len);
     }
+#elif (WIRELESS_PROTOCOL == 1) // WiFi UDP
+    int packetSize = udp.parsePacket();
+    if (packetSize) {
+        int len = udp.read(buf, sizeof(buf));
+        SERIAL.write(buf, len);
+        is_connected = true;
+        is_connected_tlast_ms = millis();
+    }
 
-#endif    
+    tnow_ms = millis(); // may not be relevant, but just update it
+    int avail = SERIAL.available();
+    if (avail <= 0) {
+        serial_data_received_tfirst_ms = tnow_ms;      
+    } else 
+    if ((tnow_ms - serial_data_received_tfirst_ms) > 10 || avail > 128) { // 10 ms at 57600 bps corresponds to 57 bytes, no chance for 128 bytes
+        serial_data_received_tfirst_ms = tnow_ms;
+
+        int len = SERIAL.read(buf, sizeof(buf));
+        udp.beginPacket(ip_udp, port_udp);
+        udp.write(buf, len);
+        udp.endPacket();
+    }
+#elif (WIRELESS_PROTOCOL == 2) // Bluetooth
+    int len = SerialBT.available();
+    if (len > 0) {
+      if (len > sizeof(buf)) len = sizeof(buf);
+      for (int i = 0; i < len; i++) buf[i] = SerialBT.read();
+      Serial.write(buf, len);
+      is_connected = true;
+      is_connected_tlast_ms = millis();
+    }
+    
+    tnow_ms = millis(); // may not be relevant, but just update it
+    int avail = SERIAL.available();
+    if (avail <= 0) {
+        serial_data_received_tfirst_ms = tnow_ms;      
+    } else {
+      if ((tnow_ms - serial_data_received_tfirst_ms) > 10 || avail > 128) { // 10 ms at 57600 bps corresponds to 57 bytes, no chance for 128 bytes
+        serial_data_received_tfirst_ms = tnow_ms;
+        int len = SERIAL.read(buf, sizeof(buf));
+        SerialBT.write(buf, len);
+      }	
+    }
```

Comment:
there is one level of {} too much
can be
```
} else
if () {
}
```

Reply:
This is intentional, as without explicit { } this is error prone according to previous experience, when editing in the future.

Reply:
can you explain, what is error prone here?

Reply:
If you add a line to else clause, one easily oversees that the brackets were missing in the first place. This is the reason, I prefer to always use brackets for else case.

Reply:
well, I don't share that view and don't see any increased error proness ... the indenting shows where one is. Editing is generally dangerous.
anyway, this shall not block it. It will naturally go eventually. :)